<ui version="4.0" >
 <class>FileProperties</class>
 <widget class="QDialog" name="FileProperties" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>344</width>
    <height>427</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>File Properties</string>
  </property>
  <widget class="QDialogButtonBox" name="buttonBox" >
   <property name="geometry" >
    <rect>
     <x>10</x>
     <y>390</y>
     <width>321</width>
     <height>32</height>
    </rect>
   </property>
   <property name="orientation" >
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons" >
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::NoButton|QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit" >
   <property name="geometry" >
    <rect>
     <x>80</x>
     <y>30</y>
     <width>252</width>
     <height>24</height>
    </rect>
   </property>
   <property name="sizePolicy" >
    <sizepolicy vsizetype="Fixed" hsizetype="Expanding" >
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox" >
   <property name="geometry" >
    <rect>
     <x>0</x>
     <y>260</y>
     <width>341</width>
     <height>127</height>
    </rect>
   </property>
   <property name="title" >
    <string>Permition</string>
   </property>
   <layout class="QGridLayout" >
    <item row="0" column="0" >
     <widget class="QLabel" name="label_9" >
      <property name="text" >
       <string>X      X</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1" >
     <widget class="QLabel" name="label_6" >
      <property name="text" >
       <string>Read</string>
      </property>
     </widget>
    </item>
    <item row="0" column="2" >
     <widget class="QLabel" name="label_7" >
      <property name="text" >
       <string>Write</string>
      </property>
     </widget>
    </item>
    <item row="0" column="3" >
     <widget class="QLabel" name="label_8" >
      <property name="text" >
       <string>Execute</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0" >
     <widget class="QLabel" name="label_10" >
      <property name="text" >
       <string>Owner</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1" >
     <widget class="QCheckBox" name="checkBox" >
      <property name="text" >
       <string/>
      </property>
     </widget>
    </item>
    <item row="1" column="2" >
     <widget class="QCheckBox" name="checkBox_2" >
      <property name="text" >
       <string/>
      </property>
     </widget>
    </item>
    <item row="1" column="3" >
     <widget class="QCheckBox" name="checkBox_3" >
      <property name="text" >
       <string/>
      </property>
     </widget>
    </item>
    <item row="2" column="0" >
     <widget class="QLabel" name="label_11" >
      <property name="text" >
       <string>Group</string>
      </property>
     </widget>
    </item>
    <item row="2" column="1" >
     <widget class="QCheckBox" name="checkBox_4" >
      <property name="text" >
       <string/>
      </property>
     </widget>
    </item>
    <item row="2" column="2" >
     <widget class="QCheckBox" name="checkBox_5" >
      <property name="text" >
       <string/>
      </property>
     </widget>
    </item>
    <item row="2" column="3" >
     <widget class="QCheckBox" name="checkBox_6" >
      <property name="text" >
       <string/>
      </property>
     </widget>
    </item>
    <item row="3" column="0" >
     <widget class="QLabel" name="label_12" >
      <property name="text" >
       <string>Other</string>
      </property>
     </widget>
    </item>
    <item row="3" column="1" >
     <widget class="QCheckBox" name="checkBox_7" >
      <property name="text" >
       <string/>
      </property>
     </widget>
    </item>
    <item row="3" column="2" >
     <widget class="QCheckBox" name="checkBox_8" >
      <property name="text" >
       <string/>
      </property>
     </widget>
    </item>
    <item row="3" column="3" >
     <widget class="QCheckBox" name="checkBox_9" >
      <property name="text" >
       <string/>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QGroupBox" name="groupBox_2" >
   <property name="geometry" >
    <rect>
     <x>0</x>
     <y>70</y>
     <width>341</width>
     <height>188</height>
    </rect>
   </property>
   <property name="title" >
    <string>General</string>
   </property>
   <layout class="QGridLayout" >
    <item row="0" column="0" >
     <widget class="QLabel" name="label" >
      <property name="text" >
       <string>Type:</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1" >
     <widget class="QLineEdit" name="lineEdit_2" />
    </item>
    <item row="1" column="0" >
     <widget class="QLabel" name="label_2" >
      <property name="text" >
       <string>Path:</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1" >
     <widget class="QLineEdit" name="lineEdit_3" />
    </item>
    <item row="2" column="0" >
     <widget class="QLabel" name="label_3" >
      <property name="text" >
       <string>Size:</string>
      </property>
     </widget>
    </item>
    <item row="2" column="1" >
     <widget class="QLineEdit" name="lineEdit_4" />
    </item>
    <item row="3" column="0" >
     <widget class="QLabel" name="label_4" >
      <property name="text" >
       <string>Modify Time:</string>
      </property>
     </widget>
    </item>
    <item row="3" column="1" >
     <widget class="QLineEdit" name="lineEdit_5" />
    </item>
    <item row="4" column="1" >
     <widget class="QLineEdit" name="lineEdit_6" />
    </item>
    <item row="4" column="0" >
     <widget class="QLabel" name="label_5" >
      <property name="text" >
       <string>Acess Time:</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QToolButton" name="toolButton" >
   <property name="geometry" >
    <rect>
     <x>10</x>
     <y>10</y>
     <width>60</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text" >
    <string>...</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>FileProperties</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel" >
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>FileProperties</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel" >
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
